apps/api/src/domain/games/memory-match/queries/memory-match.controller.ts

```diff
@@ -19,14 +29,20 @@ import { MemoryMatchService } from '../services/memory-match.service';
 export class MemoryMatchController {
     constructor(private readonly memoryMatchService: MemoryMatchService) {}
 
+    @ApiBearerAuth('JWT')
+    @UseGuards(OptionalJwtAuthGuard)
+    @ApiParam(buildByIdApiParamMetadata())
     @Get(`/:id`)
-    async fetchById(@Param('id') id: string) {
-        return this.memoryMatchService.fetchById(id);
+    async fetchById(@Param('id') id: string, @Request() req) {
+        return this.memoryMatchService.fetchById(id, req.user || undefined);
     }
 
+    @ApiBearerAuth('JWT')
+    @UseGuards(OptionalJwtAuthGuard)
+    @ApiParam(buildByIdApiParamMetadata())
     @Get('')
-    async fetchMany() {
+    async fetchMany(@Request() req) {
         // TODO[https://coscrad.atlassian.net/browse/CWEBJIRA-305] send back unpublished rounds to admin users in the future
-        return this.memoryMatchService.fetchMany();
+        return this.memoryMatchService.fetchMany(req.user || undefined);
```

Comment:
I've thought about injecting a `PublicUser` that's hard-wired to have no privileges and a bogus ID, but let's wait to investigate that.